server/paymentProviders/stripe/creditcard.js

```diff
@@ -197,22 +196,31 @@ const createChargeAndTransactions = async (hostStripeAccount, { order, hostStrip
   const hostFee = await getHostFee(order, host);
   const hostFeeInHostCurrency = Math.round(hostFee * hostCurrencyFxRate);
 
+  const fees = extractFees(balanceTransaction, balanceTransaction.currency);
+
+  let platformTipInHostCurrency, platformFeeInHostCurrency;
+  const platformTip = getPlatformTip(order);
+  if (platformTip) {
+    platformTipInHostCurrency = isSharedRevenue ? platformTip * hostCurrencyFxRate || 0 : fees.applicationFee;
```

Comment:
It was there from before, but looks to me like this one should be rounded

```suggestion
    platformTipInHostCurrency = isSharedRevenue ? Math.round(platformTip * hostCurrencyFxRate) || 0 : fees.applicationFee;
```

Reply:
Added it back.

Reply:
Sorry, my feedback was not very clear: the `Math.round` was not there in the previous version. Anyway, I think it's a good addition.